src/spdx_tools/spdx3/writer/json_ld/context.json

```diff
@@ -1,14 +1,28 @@
 {
+    "ai": "https://spdx.org/rdf/AI/",
+    "build": "https://spdx.org/rdf/Build/",
     "core": "https://spdx.org/rdf/Core/",
+    "dataset": "https://spdx.org/rdf/Dataset/",
+    "licensing": "https://spdx.org/rdf/Licensing/",
+    "ns0": "http://www.w3.org/2003/06/sw-vocab-status/ns#",
+    "owl": "http://www.w3.org/2002/07/owl#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "security": "https://spdx.org/rdf/Security/",
+    "sh": "http://www.w3.org/ns/shacl#",
     "software": "https://spdx.org/rdf/Software/",
-    "xsd": "http://www.w3.org/2001/XMLSchema/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
     "AIPackage": "ai:AIPackage",
     "Build": "build:Build",
     "Annotation": "core:Annotation",
     "AnonymousPayload": "core:AnonymousPayload",
     "Organization": "core:Organization",
     "Person": "core:Person",
+    "SoftwareAgent": "core:SoftwareAgent",
     "SpdxDocument": "core:SpdxDocument",
+    "spdxId": {
+        "@id": "core:spdxId",
+        "@type": "xsd:anyURI"
+    },
```

Comment:
`spdxId` is `@id` in json-ld

Reply:
That is, `spdxId` will probably never pop up in json-ld

Reply:
It was still included in the model as a (dead) reference, should be gone now.